dev-docs/bidders/consumable-server.md

```diff
@@ -0,0 +1,373 @@
+| `siteId`    | required | The site ID from Consumable.    | `12345` | `integer` |
+| `networkId` | required | The network ID from Consumable. | `9969`  | `integer` |
+| `unitId` | required | The unit ID from Consumable. | `987654`  | `integer` |
+| `unitName` | required | The unit name from Consumable. | `cnsmbl-unit`  | `string` |
+| `placementId` | required | the placementid from Consumable. | `0421008445828ceb46f496700a5fa65e` | `string` |
+
+## Table of contents
+
+* [Table of contents](#table-of-contents)
+* [Introduction](#introduction)
+* [Supported media types](#supported-media-types)
+* [Setup instructions to call Consumable through Prebid Server](#setup-instructions-to-call-consumable-through-prebid-server)
+  * [Call Consumable from a web browser](#call-consumable-from-a-web-browser)
+  * [Call Consumable from Prebid Mobile SDK](#call-consumable-from-prebid-mobile-sdk)
+  * [Call Consumable from CTV/long-form video environment](#call-consumable-from-ctvlong-form-video-environment)
+* [Bid request parameters](#bid-request-parameters)
+  * [Site](#site)
+  * [App](#app)
+* [Examples](#examples)
+
+<a id="introduction"></a>
+
+## Introduction
+
+Publishers can use Prebid Server in any of the following ways with Consumable. Consumable's adapter supports all of the following methods:
+
+* If you want to call Consumable from a web environment, you can use Prebid.js to call Prebid Server, and then Prebid Server uses our server-side adapter to call Consumable. This reduces workload on the browser. For set up instructions, see [Call Consumable from a web browser](#call-consumable-from-a-web-browser) section on this page.
+* In mobile apps, you can use the Prebid Mobile SDK to call Prebid Server and then Prebid Server uses our server-side adapter to call Consumable. For set up instructions, see [Call Consumable from Prebid Mobile SDK](#call-consumable-from-prebid-mobile-sdk) section on this page.
+* In CTV apps and other long-form video environments, you (or the SSAI vendor) can make a call to Prebid Server using OpenRTB, and then Prebid Server uses our server-side adapter to call Consumable. For set up instructions, see [Call Consumable from CTV/long-form video environment](#call-consumable-from-ctvlong-form-video-environment) section on this page.
+
+<a id="supported-media-types"></a>
+
+## Supported media types
+
+The following table lists the media types that Consumable supports. For information about the the Time-To-Live (TTL) for each media type, see [How Consumable counts impressions](https://kb.Consumableexchange.com/publishers/billing/how_Consumable_counts_impressions.htm) in our Knowledge Base.
+
+{: .table .table-bordered .table-striped }
+| Type      | Prebid Server support |
+| ----------- | ----------- |
+| banner      | Supported       |
+| video   | Supported, including ad pods for OTT    |
+| audio      | Supported       |
+
+<a id="call-consumable"></a>
```

Comment:
```suggestion
```

All headings can be targeted and thus no additional, empty anchor tags are needed